### PR TITLE
Security 0.9.0.7 changes

### PR DESCRIPTION
**Pull request by sujithvm**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: opendistro-elasticsearch-security.release-notes [hunk 1]
```diff
@@ -1,4 +1,9 @@
-## 2020-03-05, Version 0.9.0.6 (Current)
+## 2020-03-31, Version 0.9.0.7 (Current)
+
+- Fixed bug which caused user to lose roles on account password update
+- Performance improvement by implementing faster version of implies type perm
```
Comment (debjanibnrj) [inline opendistro-elasticsearch-security.release-notes:4]: I can't see the performance improvement as part of this PR

**Comment by vrozov** [comment]
Can it wait for PR #332?

File: opendistro-elasticsearch-security.release-notes (hunk 1 above)
Comment (sujithvm) [reply opendistro-elasticsearch-security.release-notes:4]: It is present in the branch but a release was not made for it hence updating the release notes now.

**Comment by sujithvm** [comment]
Sure @vrozov . Please let me know when it is done

File: opendistro-elasticsearch-security.release-notes (hunk 1 above)
Comment (vrozov) [reply opendistro-elasticsearch-security.release-notes:4]: Please update the list of fixes.

**Pull request merged**